Cargo.toml

```diff
@@ -20,6 +20,6 @@ serde = { version = "^1.0.126", features = ["derive"] }
 thiserror = "^1.0.26"
 toml = "^0.5.8"
 anyhow = { version = "^1.0.42", optional = true }
+tempfile = "^3.2.0"
 
 [dev-dependencies]
```

Comment:
The whole `[dev-dependencies]` section can be removed now

Reply:
Updated and removed this section.